### ISSUE TITLE
SConstruct file ignores environment PKG_CONFIG_PATH

### ISSUE THREAD

**Issue by Adjective-Object** [godotengine/godot#4161]
**Operating system or device:**
  Linux (NixOS)

**Issue description** (what happened, and what was expected):
Environment PKG_CONFIG_PATH is not inherited during environment creation in SConstruct.

Existing environment construction (line 67)
``` .python
env_base=Environment(tools=custom_tools, ENV = {'PATH' : os.environ['PATH']});
```
This creates a new environment ignoring everything in the operating system environment, apart from the path. 

It can be fixed by inheriting PKG_CONFIG_PATH from the environment

``` .python
env_base=Environment(tools=custom_tools, ENV = {
    'PATH' : os.environ['PATH'],
    'PKG_CONFIG_PATH' : os.environ['PKG_CONFIG_PATH']
});
```

**Steps to reproduce:**
run `scons platform=x11` with any dependency installed in a nonstandard location

**Comment by akien-mga** [comment]
That sounds like a sane improvement, could you make a pull request?

**Comment by akien-mga** [comment]
Fixed by #4179.
